web/js/map/rotation.js

```diff
@@ -4,7 +4,7 @@ import util from '../util/util';
 import { faIconUndoSVGDomEl, faIconRedoSVGDomEl } from './fa-map-icons';
 
 export default function MapRotate(ui, models, store) {
-  this.evts = util.events();
+  this.evts = util.events;
```

Comment:
```suggestion
```
This doesn't appear to be used.

Reply:
Nice catch!  I was worried it was being referenced on instances of this module but never went back and checked.  Looks like it's not.